Makefile

```diff
@@ -54,12 +54,12 @@ SETUP_ENVTEST_BIN := setup-envtest
 SETUP_ENVTEST := $(abspath $(TOOLS_BIN_DIR)/$(SETUP_ENVTEST_BIN)-$(SETUP_ENVTEST_VER))
 SETUP_ENVTEST_PKG := sigs.k8s.io/controller-runtime/tools/setup-envtest
 
-CONTROLLER_GEN_VER := v0.12.0
+CONTROLLER_GEN_VER := v0.10.0
```

Comment:
Are CONTROLLER_GEN_VER/CONVERSION_GEN_VER/GO_APIDIFF_VER also causing problems? I thought only go version downgrade is enough 🤔

Reply:
No, they're not. I just did a plain revert instead of creating a new version with the Go downgrade, but I'll fix this though.